backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/BitcodeEmbedding.kt

```diff
@@ -0,0 +1,41 @@
+package org.jetbrains.kotlin.backend.konan
```

Comment:
Any way to test bitcode embedding? I.e. compile some code to framework and link it with code with options forcing bitcode usage.